dpsim-models/src/Signal/DCGenerator.cpp

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2024 Institute for Automation of Complex Power Systems,
```

Comment:
Dont we want to have SPDX headers here as well?

@m-mirz

Reply:
This is consistent with the other files